src/Rewiews/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './style.scss';
+import { divSlider } from '../variables/export-img';
+
+const Rewiews = () => (
```

Comment:
export

Reply:
eslint want to export like a default  , but arrow  function need to be unnamed for default export ,so i decide to leave  usual export default